package.json

```diff
@@ -85,7 +86,7 @@
     "lodash": "4.17.11",
     "markdown-it": "8.4.1",
     "moment-timezone": "0.5.16",
-    "nightwatch": "1.0.19",
+    "nightwatch": "^0.9",
```

Comment:
?

Reply:
We are blocked as the browserstack tunnel integration doesn't work with nightwatch `^1.x`

Reply:
damn this is so annoying I'm trying to make it work anyway XD